### PR TITLE
Unmanaged Chats Work Properly and Support Metadata Syncing

### PR DESCRIPTION
**Pull request by loganallenc**
This should be ready to go. It's been tested heavily.

Joining unmanaged chats works, receiving invites to unmanaged chats works, title and descriptions work, metadata syncing works, etc

### CODE REVIEW

File: pkg/interface/chat/src/js/components/new.js [hunk 1]
```diff
@@ -220,19 +228,29 @@ export class NewScreen extends Component {
...
           <textarea
             className={idClasses}
-            placeholder="secret-chat"
+            placeholder="The coolest chat"
             rows={1}
             style={{
               resize: "none"
             }}
-            onChange={this.idChange}
+            onChange={this.titleChange}
```
Comment (matildepark) [inline pkg/interface/chat/src/js/components/new.js]: Oh, I found one. This should be descriptionChange, ha.

File: pkg/interface/chat/src/js/components/new.js [hunk 2]
```diff
@@ -38,9 +41,18 @@ export class NewScreen extends Component {
     }
   }
 
-  idChange(event) {
+  titleChange(event) {
+    let asciiSafe = event.target.value
+      .replace(/[^a-z0-9~_.-]/g, "-");
```
Comment (matildepark) [inline pkg/interface/chat/src/js/components/new.js]: This replaces capitals with `-`

Comment (loganallenc) [reply pkg/interface/chat/src/js/components/new.js]: fine by me for a first pass. if it's easy to make a better one, i'd love to see it. i'm not the best at regex

File: pkg/interface/chat/src/js/components/new.js (hunk 1 above)
Comment (loganallenc) [reply pkg/interface/chat/src/js/components/new.js]: good catch, thanks!

File: pkg/interface/chat/src/js/components/new.js (hunk 2 above)
Comment (matildepark) [reply pkg/interface/chat/src/js/components/new.js]: you could just stick a `toLowerCase()` on it before the regex then so we preserve the letters in the path name. This is the path name, right? Not the title itself?

Comment (loganallenc) [reply pkg/interface/chat/src/js/components/new.js]: Yes, path name must be ascii safe. That’s a good idea

**Pull request merged**